src/rpc/mining.cpp

```diff
@@ -961,3 +961,59 @@ UniValue getblocksubsidy(const UniValue& params, bool fHelp)
 
     return result;
 }
+
+UniValue getSidechainMerkleRoots(const UniValue& params, bool fHelp) {
+    if (fHelp || params.size() > 2)
+        throw runtime_error(
+                "getSidechainMerkleRoots transactions certificates\n"
+                "\nReturns Merkleroot and ScTxsCommitment.\n"
+                "\nArguments:\n"
+                "1. transactions         (array) Array of raw transactions (HEX format).\n"
```

Comment:
If they are optional parameters we should specify it (array, optional). If they are not we should enforce they are exactly 2 in the check above (param.size() != 2)